config/install.yaml

```diff
@@ -234,22 +221,16 @@ kind: ServiceAccount
 metadata:
   labels:
     app.kubernetes.io/component: rbac
-    app.kubernetes.io/created-by: numaplane
-    app.kubernetes.io/instance: controller-manager-sa
-    app.kubernetes.io/managed-by: kustomize
     app.kubernetes.io/name: serviceaccount
     app.kubernetes.io/part-of: numaplane
-  name: numaplane-controller-manager
+  name: numaplane-sa
   namespace: numaplane-system
```

Comment:
Do we need to remove all of the instances of `numaplane-system` namespace in `config` directory?

Reply:
Can you clarify? I thought you were extracting the namespace from the install.yaml file in the repository so that somebody could use their own namespace instead. For example, when Chandan is testing in the cluster right now, he's using a different namespace.

Reply:
It would be better to exclude the namespace object definition from the manifests, because ppl might have different approach to create namespace. Just like what we do in workflows:

```
kubectl create namespace argo
kubectl apply -n argo -f https://github.com/argoproj/argo-workflows/releases/download/v3.5.5/install.yaml
```

But it's okay to put the namespace definition in the k8s objects (int the manifests), since we prefer them to install in `numaplane-system` namespace. If anyone wants to install in a different namespace, then can simply override it in the kustomzation.yaml file.

Reply:
Okay. If that's how we're doing it with numaflow, I'm okay with making it the same I guess.